### PR TITLE
DOCS-2192 Orphan APIs

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/latest/about-wallarm/api-discovery.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Since the API Discovery module uses the real traffic as a data source, it helps
 * View most attacked APIs for the last 7 days.
 * Filter out only attacked APIs, sort them by number of hits.
 * Filter APIs that consume and carry sensitive data.
-* Find [shadow APIs](#shadow-api) - endpoints discovered by Wallarm, but absent in your specification (missing endpoints).
+* Find [shadow and orphan APIs](#orphan-api).
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:23]: the link is wrong

File: docs/latest/about-wallarm/api-discovery.md [hunk 2]
```diff
@@ -220,19 +220,23 @@ Each protected API endpoint will be highlighted with the corresponding icon in t
...
-* [Learn how to upload specifications for comparison to find shadow APIs →](../user-guides/api-specifications.md#revealing-shadow-api)
-* [Learn how to display found shadow API in the API Discovery section →](../user-guides/api-discovery.md#displaying-shadow-api)
+![!API Discovery - highlighting and filtering shadow API](../images/about-wallarm-waf/api-discovery/api-discovery-highlight-shadow-orphan.png)
+
+* [Learn how to upload specifications for comparison to find shadow and orphan APIs →](../user-guides/api-specifications.md#revealing-shadow-and-orphan-api)
+* [Learn how to display found shadow and orphan API in the API Discovery section →](../user-guides/api-discovery.md#displaying-shadow-api)
 
 As you find shadow APIs with Wallarm, you can update your specifications to include missing endpoints and further perform monitoring and security activities towards your API inventory in its full view.
 
+Found orphan APIs can be a reason to verify the applications' business logic and/or check the Wallarm traffic checking settings to understand whether traffic actually does not received or it is just not visible to the Wallarm node.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md:238]: 1. does not received?
 2. how to verify applications' business logic? what does it mean? to check that it really should not recieve traffic and delete an edpoint from the specification? or what?
 3. check the Wallarm checking settings - repetitions, read your texts attentively
 4. what does it mean - to check Wallarm node settings? which ones? how to make some conclusions based on that? what conclusions can be made?

Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-discovery.md:238]: Fixed, regarding #4 - asked the team, probably this is too much to provide such a details in this case.

Comment (MaximBashurov) [reply docs/latest/about-wallarm/api-discovery.md]: All fixed.

File: docs/latest/user-guides/api-specifications.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Uploading Your API Specifications <a href="../../about-wallarm/subscription-plans/#subscription-plans"><img src="../../images/api-security-tag.svg" style="border: none;"></a>
 
-In the **API Specifications** section of the Wallarm Console UI, you can keep your API specifications which Wallarm uses to uncover shadow APIs. This article gives an information on how to use this section.
+In the **API Specifications** section of the Wallarm Console UI, you can keep your API specifications which Wallarm uses to uncover shadow and orphan APIs. This article gives an information on how to use this section.
 
 **Administrators** and **Global administrators** can add, remove and download specifications and change settings of shadow API detection. Users of other [roles](../user-guides/settings/users.md#user-roles) can only view the list of uploaded specifications.
```
Comment (AnastasiaTWW) [inline docs/latest/user-guides/api-specifications.md:5]: again, orphan API is not mentioned. If I find one place where the mention of orphan API is missed, probably there are other the same places and there is a reason to recheck all the text

File: docs/latest/about-wallarm/api-discovery.md [hunk 4]
```diff
@@ -235,7 +235,7 @@ The API Discovery module automatically uncovers shadow and orphan APIs by compar
 
 As you find shadow APIs with Wallarm, you can update your specifications to include missing endpoints and further perform monitoring and security activities towards your API inventory in its full view.
 
-Found orphan APIs can be a reason to verify the applications' business logic and/or check the Wallarm traffic checking settings to understand whether traffic actually does not received or it is just not visible to the Wallarm node.
+The presence of orphan APIs can be a compelling reason to conduct a thorough verification process. This involves determining whether certain applications genuinely should not receive any traffic at these specific endpoints. Additionally, it is essential to inspect the Wallarm traffic checking settings to gain a comprehensive understanding of whether the traffic is truly not being received, or if it's simply not visible to the Wallarm node. This diligent examination ensures enhanced application security and robust traffic management.
```
Comment (AnastasiaTWW) [inline docs/latest/about-wallarm/api-discovery.md]: too much new words like compelling, thorough, etc. non-native speaker can just not understand that

Comment (AnastasiaTWW) [reply docs/latest/about-wallarm/api-discovery.md]: we do not use it's, we use it is, it is our style guide

Comment (AnastasiaTWW) [reply docs/latest/about-wallarm/api-discovery.md]: This diligent examination ensures enhanced application security and robust traffic management. - how? some really general staff

File: docs/latest/user-guides/api-specifications.md (hunk 3 above)
Comment (AnastasiaTWW) [reply docs/latest/user-guides/api-specifications.md]: "option is selected by default). Hourly comparison allows finding additional shadow APIs as API Discovery discovers more endpoints. Specification uploaded from URL is updated before each comparison." - find this in the text and mention orphan apis as well

Comment (MaximBashurov) [reply docs/latest/user-guides/api-specifications.md]: Fixed

**Pull request merged**